roboprop_client/export_model.py

```diff
@@ -72,7 +72,7 @@ def export_sdf(out_dir: Path, model_name: str, blend_file_path: Path):
         )
         pose = ElementTree.SubElement(link, "pose")
         pose.set("degrees", "1")
-        pose.text = "0 0 0 90 0 -90"
+        pose.text = "0 0 0 90 0 90"
```

Comment:
The inversion on yaw here brings the models back to the correct rotation.
But should we do it further up the chain? e.g by resetting center in blender before running the export?

Reply:
Thank you Tomo! Sorry that i missed this.

> But should we do it further up the chain?

Yes, in the export_fbx.py, there is the `bpy.ops.export_scene.fbx()` call with has these two options `axis_forward='-Z', axis_up='Y'` https://docs.blender.org/api/current/bpy.ops.export_scene.html#bpy.ops.export_scene.fbx 

The confusing thing is that we're currently using the defaults and the defaults are the same as in the OBJ export, so nothing should change.

If i remember correctly, all the gltf, fbx, and obj exports were oriented the same. I'll try to look into this too.